services/QuillLMS/app/services/classlink_integration/users_fetcher.rb

```diff
@@ -5,19 +5,27 @@ class UsersFetcher < BaseFetcher
     RESOURCE = 'users'
     QUERY_STRING = '?sort=sourcedId&orderBy=asc'
 
+    ROLES = {
+      'student' => 'student',
+      'teacher' => 'teacher',
+      'administrator' => 'admin'
+    }.freeze
+
     def transform_api_response(user_hash)
       {
         account_type: 'classlink',
         email: user_hash['email'],
         name: "#{user_hash['givenName']} #{user_hash['middleName']} #{user_hash['familyName']}".squish,
-        password: generate_password(user_hash['familyName'], user_hash['givenName']), # TODO: remove after SSO linkage
-        role: user_hash['role'],
+        password: nil,
+        role: self.class.coerce_role(user_hash['role']),
```

Comment:
This feel like a passthrough method.  What about using ROLES directly here:

        role: ROLES[user_hash['role'])